trace-java-client/src/main/java/Test.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import com.amazon.opentelemetry.trace.client.JaegerTraceEmitClient;
+import com.amazon.opentelemetry.trace.client.ZipkinTraceEmitClient;
+
+public class Test {
+
+  public static void main(String[] args) throws Exception {
+
+//    emitJaegerTraces();
```

Comment:
shouldn't it read some arg and determine which emit to run?

Reply:
it's for developer to do local testing/debugging. no one reference it. just keep it simple

Reply:
It might be better to put it into the test package, or just add some comment to `Test` class.